client/client.go

```diff
@@ -963,3 +984,93 @@ func (r *NotaryRepository) DeleteTrustData() error {
 	}
 	return nil
 }
+
+// ListRootCerts returns current trusted root certificates.
+// (In particular, not old certificates which we have already
+// rotated from but are still using for signing to allow rollover,
+// even if they are still trusted locally.)
+func (r *NotaryRepository) ListRootCerts() ([]*x509.Certificate, error) {
+	_, err := r.Update(false)
+	if err != nil {
+		return nil, err
+	}
+
+	keyIDs := r.tufRepo.Root.Signed.Roles[data.CanonicalRootRole].KeyIDs
```

Comment:
Ah yep, that true. There was something I couldn't grasp in the back of my mind telling me we couldn't rely on what's on disk and I think you've just worked it out. Also, with @riyazdf's work on cert pinning we can start looking at removing the cert store entirely and just relying on what's in root.json.